raiden/tests/scenarios/ci/sp1/pfs6_simple_path_rewards.yaml

```diff
@@ -54,6 +54,8 @@ scenario:
 
             # Node3 cannot send tokens to node0 due to imbalanced channels
             # We expect the pfs not to send any route, but to charge for the information
+            # but the node can check if direct channel is usable so we need a deposit
```

Comment:
`if direct channel is` -> `if a/the direct channel is`

Reply:
thanks, done